pkg/doc/verifiable/credential.go

```diff
@@ -185,7 +185,29 @@ const defaultSchema = `{
 
 const jsonSchema2018Type = "JsonSchemaValidator2018"
 
-const vpType = "VerifiablePresentation"
+const (
+	baseContext = "https://www.w3.org/2018/credentials/v1"
```

Comment:
It would be good to add a comment linking to the spec.
https://www.w3.org/TR/vc-data-model/#base-context

Reply:
Added the links.